small updates to changelog (#1639)
stripe/stripe-python: CHANGELOG.md

## 13.0.1 - 2025-10-03
* [#1626](https://github.com/stripe/stripe-python/pull/1626) Make the new nested params classes correctly importable

>   - For example, `from stripe.params.checkout import SessionCreateParamsDiscount` would raise an error when it shouldn't have. This is fixed.

Reviewer: nit: 
```suggestion
  - For example, `from stripe.params.checkout import SessionCreateParamsDiscount` would raise an error in `13.0.0` when it shouldn't have. This is fixed.
```

>   - For example, `from stripe.params.checkout import SessionCreateParamsDiscount` would raise an error in `13.0.0` when it shouldn't have. This is fixed.

Reviewer: Another alternate you could consider for 
```suggestion
  - For example, In SDK `v13.0.0`, `from stripe.params.checkout import SessionCreateParamsDiscount` would raise an error when it shouldn't have. This is fixed.
```

  - reported in https://github.com/stripe/stripe-python/issues/1625
* [#1633](https://github.com/stripe/stripe-python/pull/1633) Add new `async` optional dependency
  - Now, `pip install stripe[async]` gets you everything you need to make async HTTP calls out of the box.

>   - reported in https://github.com/stripe/stripe-python/issues/1558

Reviewer: nit:
```suggestion
  - Reported in https://github.com/stripe/stripe-python/issues/1558
```

* [#1627](https://github.com/stripe/stripe-python/pull/1627) Re-add `tests` to our `sdist`

>   - reported in https://github.com/stripe/stripe-python/issues/1616

Reviewer: nit: 
```suggestion
  - Reported in https://github.com/stripe/stripe-python/issues/1616
```

> * [#1632](https://github.com/stripe/stripe-python/pull/1632) remove gpg step from publish. PyPI has ignored these for years, so we took the opportunity to simplify our build process

Reviewer: We can remove this from the changelog as this is not a user facing change.

Author: I figured it was worth mentioning in case someone is looking for it, even if it's not actively used.
